consensus/spos/bls/subroundBlockV2.go

```diff
@@ -0,0 +1,75 @@
+package bls
+
+import (
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/common"
+	"github.com/ElrondNetwork/elrond-go/consensus"
+	"github.com/ElrondNetwork/elrond-go/consensus/spos"
+	"golang.org/x/net/context"
+)
+
+type subroundBlockV2 struct {
+	*subroundBlock
+}
+
+// NewSubroundBlockV2 creates a subroundBlockV2 object
+func NewSubroundBlockV2(subroundBlock *subroundBlock) (*subroundBlockV2, error) {
```

Comment:
also here. Are we gonna use the v2 on the regular elrond-go node?

Reply:
Will be used also in the main chain